doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -380,7 +380,7 @@ Drivers
 
 This section provides detailed lists of changes by :ref:`driver <drivers>`.
 
-|no_changes_yet_note|
+* Added a new page describing the :ref:`sQSPI MSPI shim <mspi_sqspi>` driver.
```

Comment:
```suggestion
* Added the :ref:`mspi_sqspi` that allows for communication with devices that use MSPI bus-based Zephyr drivers.
```
Now that I look this closer, the PR actually adds just the driver doc and the driver itself was added earlier. However, I believe adding the driver was not mentioned earlier in the changelog, so this entry could be here.
@annwoj opinions?

Reply:
That is correct. If it wasn't mentioned earlier we can definitely mention it here. I'm fine with the suggestion

Reply:
Applied.